src/test/unittest/unittest.sh

```diff
@@ -2797,7 +2797,9 @@ if [ "$CLEAN_FAILED_REMOTE" == "y" ]; then
 	for ((i=0;i<=$NODES_ALL;i++));
 	do
 		N[$i]=${NODE_WORKING_DIR[$i]}/$curtestdir/data/
-		run_command ssh $SSH_OPTS ${NODE[$i]} touch ${N[$i]}nomatch; rm -rf ${N[$i]}*
+		if [ -n "${N[$i]}" ]; then
+			run_command ssh $SSH_OPTS ${NODE[$i]} "rm -rf ${N[$i]}*"
```

Comment:
I'd prefer to just "exit 1" when N[$i] is empty.

Also, please verify that ${NODE_WORKING_DIR[$i]} and $curtestdir are not empty.

Reply:
Actually N[$i] can't be empty because it always appends /data/.